.github/CODEOWNERS

```diff
@@ -416,7 +401,6 @@ app/serializers/prescription_preference_serializer.rb @department-of-veterans-af
 app/serializers/prescription_serializer.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/vsa-ebenefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/serializers/profile_photo_attachment_serializer.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/serializers/rated_disabilities_serializer.rb @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/serializers/rating_info_serializer.rb @department-of-veterans-affairs/lighthouse-banana-peels
```

Comment:
I don't see any of these files and folders within `app/serializers` (except the `app/serializers/lighthouse` line)  being replaced with any top-level ownership. If lighthouse-banana-peels doesn't own them, please replace it with `@department-of-veterans-affairs/backend-review-group`